qubes-artwork.spec.in

```diff
@@ -9,11 +9,6 @@ Vendor:		Invisible Things Lab
 URL:		https://www.qubes-os.org
 BuildArch:	noarch
 
-Obsoletes:	system-logos
-Obsoletes:	redhat-logos
-Obsoletes:	fedora-logos
-Obsoletes:	desktop-backgrounds-compat
```

Comment:
This looks like unrelated change, but also I think this part may be not needed anymore. Lets see, if something will break after removing it, we can always restore it.

Reply:
Turns out it was needed: https://openqa.qubes-os.org/tests/overview?distri=qubesos&version=4.3&build=4.3.202502182300&groupid=1